### PR TITLE
Added UI elements from Photogram Industrial 3 exercise

### PR DESCRIPTION
**Pull request by brvarner**

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
       <div class="container-fluid">
         <%= link_to 'Photogram', root_path, class: "navbar-brand" %>
         <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
          <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-        <li class="nav-item">
-          <%= link_to 'Photos', root_path , class: "nav-link active" %>
+         <ul class="navbar-nav mr-auto">
```
Comment (AdolfoNava) [inline app/views/shared/_navbar.html.erb]: Left mr-auto still in the class attribute

**Review by AdolfoNava — APPROVED** [review]
Looks Good to me

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,8 +12,15 @@
   </head>
 
   <body>
+    <%= render 'shared/navbar' %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:15]: nice job adding your navbar and cdn assets and flash messages to a partial! this helps clean up the code alot

File: app/views/photos/_photo.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div id="<%= dom_id photo %>">
   <p>
     <strong>Image:</strong>
-    <%= photo.image %>
+    <%= image_tag photo.image %>
```
Comment (JoseMDO) [inline app/views/photos/_photo.html.erb:4]: great job using helper methods

File: app/views/shared/_navbar.html.erb [hunk 4]
```diff
@@ -1,34 +1,40 @@
...
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-        <li class="nav-item">
-          <%= link_to 'Photos', root_path , class: "nav-link active" %>
+         <ul class="navbar-nav">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:15]: remember we can use helper methods here like link_to

File: app/views/shared/_navbar.html.erb [hunk 5]
```diff
@@ -1,34 +1,40 @@
...
+            <a class="dropdown-item" href="/comments">Comments</a>
+            <a class="dropdown-item" href="/follow_requests">Follow Requests</a>
+            <a class="dropdown-item" href="/likes">Likes</a>
+          </div>
         </li>
+      </ul>
         <% if user_signed_in? %>
 <li class="nav-item">
-      <%= link_to "Edit profile", edit_user_registration_path(current_user), class: "nav-link" %>
+      <%= link_to "Edit #{current_user.username}", edit_user_registration_path(current_user), class: "nav-link" %>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:24]: Nice!

File: lib/tasks/dev.rake [hunk 6]
```diff
@@ -10,21 +10,32 @@ task sample_data: :environment do
     User.destroy_all
   end
 
+  usernames = Array.new { Faker::Name.first_name }
+
+  usernames << "alice"
+  usernames << "bob"
+
+  usernames.each do |username|
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:18]: Great job on sample data!

**Review by JoseMDO — APPROVED** [review]
Great job Brandon!

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (brvarner) [reply app/views/layouts/application.html.erb:15]: Thank you!

File: app/views/photos/_photo.html.erb (hunk 3 above)
Comment (brvarner) [reply app/views/photos/_photo.html.erb:4]: Thanks Jose!

File: app/views/shared/_navbar.html.erb (hunk 4 above)
Comment (brvarner) [reply app/views/shared/_navbar.html.erb:15]: Ooo very good point, I'll make that change!

File: app/views/shared/_navbar.html.erb (hunk 5 above)
Comment (brvarner) [reply app/views/shared/_navbar.html.erb:24]: Thank you!

File: lib/tasks/dev.rake (hunk 6 above)
Comment (brvarner) [reply lib/tasks/dev.rake:18]: Thanks Jose!

**Pull request merged**